js/persistent-notification-container.js

```diff
@@ -0,0 +1,157 @@
+(function ($) {
+  'use strict';
+
+  let getMenuItemLnk = function (item) {
+    return $(item).find('> a');
+  },
+
+  focusFirstMenuItem = function ($element) {
+    $element.find('a:first').trigger('focus');
+  },
+
+  removeActiveClasses = function ($menuItems) {
+    $.each($menuItems, function (idx, item) {
+      if (getMenuItemLnk(item).is('.pf-is-active')) {
+        getMenuItemLnk(item).removeClass('pf-is-active');
+      }
+    });
+  },
+
+  removeAriaCurrent = function ($menuItems) {
+    $.each($menuItems, function (idx, item) {
+      if (getMenuItemLnk(item).attr('aria-current') === 'true') {
+        getMenuItemLnk(item).attr('aria-current', false);
+      }
+    });
+  },
+
+  popNotification = function ($element) {
+    $('main > .notification-container').append($element);
+
+    setTimeout(function () {
+      $element.remove();
+    }, 6000);
+  },
+
+  alert1NotifMarkup = function () {
+    return `
+      <div
+        id="alert-notification"
+        class="pf-c-toast pf-is-warning">
+        <div role="alert">
```

Comment:
The [container in the html](https://github.com/andresgalante/jenn/pull/17/files#diff-f50cf1eb4c2ae7810fcf95234ff5bc17R51) includes
`<div class="notification-container" aria-live="polite">`

But `role="alert"` also [includes an implicit `aria-live="assertive`](https://www.w3.org/TR/wai-aria-practices-1.1/examples/alert/index.html).

Should we include one or two of these alerts with out using `role="alert"` just to test how they might behave differently?